test/packages/good/data_stream/foo/manifest.yml

```diff
@@ -1,6 +1,5 @@
 title: Nginx access logs
 type: logs
-release: experimental
```

Comment:
Is the release tag is removed everywhere now, is there a way to test the "deprecation" part?

Reply:
The release tag is still present in other packages being tested, I only removed it from the "good" one.